api/v1/helper.go

```diff
@@ -121,7 +121,7 @@ func IsSupportedModel(vendorID, deviceID string) bool {
 			return true
 		}
 	}
-	log.Info("IsSupportedModel():", "Unsupported model:", "vendorId:", vendorID, "deviceId:", deviceID)
+	glog.Infof("IsSupportedModel(): Unsupported model: vendorId=%s, deviceId=%s", vendorID, deviceID)
```

Comment:
we really need everything to use container runtime logger, as this  helper is used by the controller as well, we dont want to use two different loggers.

maybe we need to tackle  switching config daemon to controller runtime logger as a separate PR ?

Reply:
and in that separate PR, finally close https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/480 ;)

Reply:
#527 addresses most issues. #480 was updated to reflect remaining work.